src/products/test/integ/json_schemas.json

```diff
@@ -0,0 +1,125 @@
+{
+  "/products/all": {
+    "type": "array",
+    "data": {
```

Comment:
I believe it should be `items` here?
Could you double check if the "jsonschema.validate()" will fail if you modify  the schema?